src/Evolution/DiscontinuousGalerkin/SlopeLimiters/WenoHelpers.hpp

```diff
@@ -4,13 +4,28 @@
 #pragma once
 
 #include <cstddef>
+#include <unordered_map>
```

Comment:
Can you make the commit message more informative please?

Reply:
You haven't told me what you want clarified. Would it be helpful to add a sentence like "Adds a helper function that computes the WENO reconstruction as the weighted sum of different input polynomials" to the commit message?